### ISSUE TITLE
Player Rollback

### ISSUE THREAD

**Issue by DoubleBrackets** [GingerWalnut/SQBeyondPublic#139]
So I was doing normal stuff, when I had to go. I logged out in one of my farms, and I when I came back, everything was fine. I played for a bit, then logged off again. When I logged on, however, my player was rolled back to when I logged out the first time, in the farm. I figured it was a one time bug, and continued playing for a bit. However, when relogged a while later, the same thing happened. I have gotten a mod to remove the dup'ed items, but I am unable to do anything since I just get rolled back.(Mcmmo gets rolled back too). To my knowledge, it is still happening whenever I log on.
IGN: DoubleBrackets

**Comment by StarWarden** [comment]
I had a similar thing happen to me last night. So I logged off after parking and unpiloting my ship on the ground on Jurion in the Arator System. When I logged back in later, I was floating in space way over in the Yavar System. I had been flying there earlier in the day. My contracts and money were not rolled back, but my location and inventory were. I had dropped off my inventory at my void station on the way to Jurion. As far as I know my ship is still on Jurion and the items in my inventory may be duped. I didn't check my Mcmmo.
IGN: ThunderStryken
